Doc/library/os.rst

```diff
@@ -197,6 +197,10 @@ process and user.
    changes to the environment made by :func:`os.putenv`, by
    :func:`os.unsetenv`, or made outside Python in the same process.
 
+   On Windows, :func:`get_user_default_environ` can be used to update
+   :data:`os.environ` to the latest user and system environment variables, such
+   as the ``Path`` variable.
```

Comment:
Normally, a new environment block gets passed to a spawned child process, such as via the `env` parameter of `subprocess.Popen`. I'd emphasize that over updating `os.environ`, which seems to be a point of controversy.